versions.yaml

```diff
@@ -19,5 +19,5 @@ libraries:
 
   qat:
     url: "https://01.org/sites/default/files/downloads"
-    version: "1.7.l.4.5.0-00034"
-    sha256: "c42a3afc1a5c76d441eaca8b97dc1f9ee64939ec001539ee1a2f3b39b7543c8e"
+    version: "1.7.l.4.6.0-00025"
```

Comment:
@ipuustin when the driver is configured `./configure --flags` in Docker files or ansible files are called with `--enable-qat-uio`, but in this version that flag seems to be removed, a WARNING is shown, do you think it will be better if that flag is removed?

Reply:
Good catch! Those flags are not used anymore in qat compilation. I added a patch which removes them.